src/nnode_training_edits.jl

```diff
@@ -0,0 +1,32 @@
+
+using NeuralPDE, OrdinaryDiffEq, DiffEqFlux, OptimizationPolyalgorithms, Lux, Random, Flux, OptimizationOptimJL, Test, Statistics
```

Comment:
what's this? Is this supposed to be a test script?

Reply:
Yes, I will remove it in the next commit

Reply:
Removed